resources/views/front-page.blade.php

```diff
@@ -1,8 +1,185 @@
+{{--
```

Comment:
Template name isn't need if it's using `front-page`

Reply:
So I want to rename something less "Front Page" based since essentially they could reuse this on other pages too

Reply:
Just trying to understand your thinking here... Why would we want to use the front-page on other pages?

Reply:
Audience pages are usually a carbon copy of home page template with just allowing editors to select new slides and category of posts/books/events etc